tools/klee-stats/klee-stats

```diff
@@ -194,6 +194,8 @@ def grafana(dirs):
         startTime = os.path.getmtime(dr)
         fromTime = frm - startTime if frm - startTime > 0 else 0
         toTime = to - startTime if to - startTime > fromTime else fromTime + 100
+        #convert to microseconds
+        startTime, fromTime, toTime = startTime*1000000, fromTime*1000000, toTime*100000
```

Comment:
0 missing for `toTime`?

Reply:
Good catch.

Reply:
In Python3 you can use `1_000_000` notation... might help here.